google/cloud/pubsub/samples/pubsub_samples_common.cc

```diff
@@ -63,7 +63,7 @@ google::cloud::testing_util::Commands::value_type CreateSubscriberCommand(
     google::cloud::pubsub::Subscriber client(
         google::cloud::pubsub::MakeSubscriberConnection(subscription));
```

Comment:
It is worth `move`ing the `subscription` now (or not giving it a name)?

Reply:
Doh, fixed.